tv/pages/rooms/[id].tsx

```diff
@@ -1,17 +1,18 @@
 import { getSessions } from '../../api/queries/getSessions';
 import { getRooms } from '../../api/queries/getRooms';
-import { Session } from '../../interfaces';
+import { Session } from '../../interfaces/session';
+import { Params } from '../../interfaces';
 import RoomDisplay from '../../components/RoomDisplay';
 
-type RoomProps = {
+type RoomsProps = {
   sessions: Session[];
 };
 
-export declare type Params = {
-  [param: string]: any;
+export declare type RoomsParams = {
+  [param: string]: Params;
 };
 
-export default function Room(props: RoomProps) {
+export default function Rooms(props: RoomsProps) {
```

Comment:
As this is an [id] page to display an individual item, the function and types names should be singular. Room, RoomParams, RoomProps.

Reply:
Fixed